Lib/test/test_traceback.py

```diff
@@ -777,23 +796,16 @@ def f():
         ]
         self.assertEqual(actual, expected)
 
+
 @cpython_only
 @requires_debug_ranges()
-class CPythonTracebackErrorCaretTests(TracebackErrorLocationCaretTests):
+class CPythonTracebackErrorCaretTests(
+    CAPIExceptionFormattingMixin,
+    TracebackErrorLocationCaretTests,
+):
```

Comment:
I think it can be a bit tidier if it would follow the pattern of PyExcReportingTests / CExcReportingTests where the c mixin doesn't subclass the python mixin.